pages/vi/profiles/cquarte1.md

```diff
@@ -0,0 +1,30 @@
+**Carlos Quarterman**
+#Albany, GA
```

Comment:
Looks like adding address as header is breaking how it displays.
Can you please fix this.
Please check https://raw.githack.com/cquarte1/cquarter.github.io/CarlosVI/#!pages/vi/profiles/cquarte1.md to make sure your changes are as you expect it to be.